src/Sulu/Bundle/ContactBundle/DataFixtures/ORM/LoadDefaultTypes.php

```diff
@@ -27,92 +27,91 @@ class LoadDefaultTypes extends AbstractFixture implements OrderedFixtureInterfac
 
+        $emailType1 = new EmailType();
+        $emailType1->setId(1);
+        $emailType1 = $manager->merge($emailType1);
         $emailType1->setName('email.work');
-        $manager->persist($emailType1);
 
         $this->addReference('email.type.work', $emailType1);
 
         $emailType2 = new EmailType();
         $emailType2->setId(2);
+        $emailType2 = $manager->merge($emailType2);
         $emailType2->setName('email.home');
-        $manager->persist($emailType2);
 
         $this->addReference('email.type.home', $emailType2);
 
-        $addressType1 = new AddressType();
-        $addressType1->setId(1);
 
-        // force id = 1
-        $metadata = $manager->getClassMetaData(get_class($addressType1));
+        // address types
+        $metadata = $manager->getClassMetaData(AddressType::class);
         $metadata->setIdGeneratorType(\Doctrine\ORM\Mapping\ClassMetadata::GENERATOR_TYPE_NONE);
 
+        $addressType1 = new AddressType();
+        $addressType1->setId(1);
+        $addressType1 = $manager->merge($addressType1);
         $addressType1->setName('address.work');
-        $manager->persist($addressType1);
 
         $addressType2 = new AddressType();
         $addressType2->setId(2);
+        $addressType2 = $manager->merge($addressType2);
         $addressType2->setName('address.home');
-        $manager->persist($addressType2);
 
-        $urlType1 = new UrlType();
-        $urlType1->setId(1);
 
-        // force id = 1
-        $metadata = $manager->getClassMetaData(get_class($urlType1));
+        // url types
+        $metadata = $manager->getClassMetaData(UrlType::class);
         $metadata->setIdGeneratorType(\Doctrine\ORM\Mapping\ClassMetadata::GENERATOR_TYPE_NONE);
 
+        $urlType1 = new UrlType();
+        $urlType1->setId(1);
+        $urlType1 = $manager->merge($urlType1);
         $urlType1->setName('url.work');
-        $manager->persist($urlType1);
 
         $urlType2 = new UrlType();
         $urlType2->setId(2);
+        $urlType2 = $manager->merge($urlType2);
         $urlType2->setName('url.home');
-        $manager->persist($urlType2);
-
-        $manager->flush();
 
-        $faxType1 = new FaxType();
-        $faxType1->setId(1);
 
-        // force id = 1
-        $metadata = $manager->getClassMetaData(get_class($faxType1));
+        // fax types
+        $metadata = $manager->getClassMetaData(FaxType::class);
         $metadata->setIdGeneratorType(\Doctrine\ORM\Mapping\ClassMetadata::GENERATOR_TYPE_NONE);
 
+        $faxType1 = new FaxType();
+        $faxType1->setId(1);
+        $faxType1 = $manager->merge($faxType1);
         $faxType1->setName('fax.work');
-        $manager->persist($faxType1);
 
         $faxType2 = new FaxType();
         $faxType2->setId(2);
+        $faxType2 = $manager->merge($faxType2);
         $faxType2->setName('fax.home');
-        $manager->persist($faxType2);
 
+
+        // save
```

Comment:
Remove these two lines (two lines in a row, and the comment doesn't add any real value)